mne/preprocessing/ica.py

```diff
@@ -1195,16 +1201,17 @@ def find_bads_ecg(self, inst, ch_name=None, threshold=None, start=None,
 
     @verbose
     def find_bads_ref(self, inst, ch_name=None, threshold=3.0, start=None,
-                      stop=None, l_freq=None, h_freq=None,
-                      reject_by_annotation=True, verbose=None):
+                      stop=None, l_freq=None, h_freq=None, method='together',
+                      reject_by_annotation=True, bad_measure="zscore",
```

Comment:
bad_measure -> measure? function already has bads in its name

Reply:
Sure. Should I also extend this option to EOG and ECG?

Reply:
+1 to do it for consistency